t/plugin/log-rotate3.t

```diff
@@ -132,3 +132,40 @@ start xxxxxx
     }
 --- response_body
 passed
+
+
+=== TEST 4: max_kept effective on differently named compression files
```

Comment:
Could you add a test case with enable_compression = false because the issue you fixed is like this

Reply:
done